Extra/Pixlr.php

```diff
@@ -24,32 +24,59 @@
 
 class Pixlr
 {
+    /**
+     * @var string
+     */
     protected $referrer;
 
+    /**
+     * @var string
+     */
     protected $secret;
 
+    /**
+     * @var MediaManagerInterface
+     */
     protected $mediaManager;
 
+    /**
+     * @var RouterInterface
+     */
     protected $router;
 
+    /**
+     * @var Pool
+     */
     protected $pool;
 
+    /**
+     * @var EngineInterface
+     */
     protected $templating;
 
+    /**
+     * @var ContainerInterface
+     */
     protected $container;
 
+    /**
+     * @var string[]
+     */
     protected $validFormats;
 
+    /**
+     * @var string
```

Comment:
... but is a string ;)
https://github.com/sonata-project/SonataMediaBundle/blob/master/Extra/Pixlr.php#L65